.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: c
+addons:
+  apt:
+    packages:
+      libpulse-dev
 script:
   - ./autogen.sh
-  - ./configure
+  - ./configure --prefix=/usr
```

Comment:
Can you say why this change is necessary? Please also include your motivation in the commit message.

Reply:
Apparently it isn't.

Reply:
Great. Thanks for fixing.